### DISCUSSION TITLE
Is there any method or feature available in the pakage for set a custom eastern timestamp

### DISCUSSION THREAD

**Discussion by Naveenzerobugz** [snowplow/dbt-snowplow-web#220] [General]
To display the dashboard in Eastern Time instead of UTC, I need to adjust the timezone setting to switch it to Western Time Zone.
can you help me setup a custom timestamp? this package 
like this 'snowplow:timezone': 'America/New_York'

**Comment by rlh1994** [comment]
All Snowplow tracking, and by extension all data modelling our packages, stick quite strictly to using the UTC timezone for all our timestamp fields to avoid timezone conflicts and confusion. It's unlikely that we would deviate from this as most warehouses and BI tools offer some functionality to view timestamps in a given timezone based on a setting/connection information (I know Tableau doesn't and has been requested for 10 years, but _most_ do). 

I'll leave the issue and this discussion open for any further input in case we do decide to change this approach in the future, but for now I would explore the ability of your dashboarding tool to how you might alter timezones presented of data in there if possible.
